### DISCUSSION TITLE
posts/decade-of-vim/

### DISCUSSION THREAD

**Discussion by giscus** [nithinbekal/nithinbekal.github.io#31] [Comments]
# posts/decade-of-vim/

Nithin Bekal's blog about programming - Ruby, Rails, Vim, Elixir.

https://nithinbekal.com/posts/decade-of-vim/?ref=dailydev

**Comment by alexkubica** [comment]
Thank you for writing this post!
How did you set neovim integration?

**Reply by nithinbekal** [reply]
@alexkubica I used [kickstart.nvim](https://github.com/nvim-lua/kickstart.nvim) to get started with my config file, and kept adding things as I needed them. My current config is [here](https://github.com/nithinbekal/dotfiles/blob/master/.config/nvim/init.lua).
